tests/data/layout2.json

```diff
@@ -6,13 +6,22 @@
         "path": "derivatives"
     },
     "dpath_sourcedata": {
+        "path": "scratch"
+    },
+    "dpath_src_tabular": {
+        "path": "scratch/tabular"
+    },
+    "dpath_src_imaging": {
+        "path": "scratch/imaging"
+    },
+    "dpath_to_bidsify": {
         "path": "dicom"
     },
     "dpath_downloads": {
         "path": "downloads"
```

Comment:
sourcedata/downloads?

Reply:
Leaving these paths as they are (except for `code` -> `proc`) since I wanted to try with a few different layouts for the tests